web_advanced_search_wildcard/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+# Copyright 2016 Therp BV <http://therp.nl>.
+# Copyright 2017 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Wildcard in advanced search",
+    "summary": "Webmodule to add wildcard operators in advanced search field",
+    "version": "10.0.1.0.0",
+    "category": "web",
+    "license": "AGPL-3",
+    "author": "initOS GmbH & Co. KG,Odoo Community Association (OCA),Therp BV",
```

Comment:
shouldn't OCA stays always at the end? :thinking:

Reply:
@simahawk, just changed!